tests/model/test_model.py

```diff
@@ -2407,6 +2483,22 @@ def test_is_muted_stream(
         model.muted_streams = muted_streams
         assert model.is_muted_stream(stream_id) == is_muted
 
+    @pytest.mark.parametrize(
+        ["desktop_notified_streams", "stream_id", "is_enabled"],
+        [
+            ({1}, 1, True),
+            ({1, 3, 7}, 2, False),
+            (set(), 1, False),
+        ],
+        ids=["single_stream", "multiple_streams", "no_stream"],
```

Comment:
The ids don't speak about the test case but what's the state of `desktop_notified_streams`. For instance, it would be better to use "notification_enabled" or something along those lines for the first id.